packages/stepper/package.json

```diff
@@ -0,0 +1,115 @@
+{
+  "name": "@talend/react-stepper",
+  "description": "Provide a stepper component",
+  "main": "lib/index.js",
+  "mainSrc": "src/index.js",
+  "license": "Apache-2.0",
+  "scripts": {
+    "prepublish": "babel -d lib ./src/ && rimraf lib/**/*.test.js && cpx -v \"./src/**/*.scss\" ./lib",
+    "start": "start-storybook -p 6009",
+    "test": "jest --silent",
+    "test:noisy": "jest",
+    "test:watch": "jest --watch",
+    "test:cov": "jest --coverage",
+    "test:demo": "build-storybook",
+    "lint:es": "eslint --config ../../.eslintrc src",
+    "lint": "npm run lint:es",
+    "storybook": "start-storybook -p 6009",
+    "build-storybook": "build-storybook"
+  },
+  "keywords": [
+    "react",
+    "talend",
+    "stepper"
+  ],
+  "author": "Talend Frontend <frontend@talend.com>",
+  "homepage": "https://github.com/Talend/ui/tree/master/packages/stepper#readme",
+  "bugs": {
+    "url": "https://github.com/Talend/ui/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Talend/ui.git"
+  },
+  "dependencies": {
+    "classnames": "^2.2.5",
+    "invariant": "^2.2.2",
+    "keycode": "^2.2.0",
+    "lodash": "^4.17.4"
+  },
+  "devDependencies": {
+    "@babel/cli": "^7.2.0",
+    "@babel/core": "^7.2.0",
+    "@babel/plugin-proposal-class-properties": "^7.2.0",
+    "@babel/plugin-proposal-object-rest-spread": "^7.2.0",
+    "@babel/polyfill": "^7.0.0",
+    "@babel/preset-env": "^7.2.0",
+    "@babel/preset-react": "^7.0.0",
+    "@storybook/addon-a11y": "^5.1.0",
+    "@storybook/addon-actions": "^5.1.0",
+    "@storybook/addons": "^5.1.0",
+    "@storybook/react": "^5.1.0",
+    "@talend/icons": "^4.4.0",
+    "@talend/react-components": "^4.4.0",
```

Comment:
Can you put all `@talend` packages as dep (not dev-dep) please ?

Reply:
No because it's managed by the final app, isn't it?